### ISSUE TITLE
On Sequential() LSTM, .evaluate not outputting results

### ISSUE THREAD

**Issue by cjmcmurtrie** [fchollet/keras#552]
Hi there,

First of all I should mention that Keras is brilliant. Keep up the great work.

I'm training two stacked LSTM layers for a vector-2-vector mapping. The model is:

    model = Sequential()
    model.add(LSTM(embedding_size, hidden_size, activation='sigmoid', truncate_gradient=-1, return_sequences=True))
    model.add(LSTM(hidden_size, output_size, activation='sigmoid', truncate_gradient=-1, return_sequences=True))
    model.compile(loss='mse', optimizer='adam')

Training the model works fine. For one epoch:

      Epoch 0
      9500/9500 [==============================] - 125s - loss: 0.0054 - acc: 0.2139 - val_loss: 0.0038 - val_acc: 0.0475

However, the evaluation code:

    model.evaluate(Xtest, Ytest, batch_size=batch_size, show_accuracy=True, verbose=1)

Does not output anything informative:

      Evaluation...
      401/401 [==============================] - 1s    

The above is all that happens before ending, without any loss/accuracy values.

Any suggestions?

Thanks!

**Comment by fchollet** [comment]
`evaluate` does not print the loss / accuracy (maybe it should, though). Instead, it returns it (as a scalar if `show_accuracy=False`, else as a `(loss, acc)` tuple). So just `print` the output of `evaluate` instead.
